packages/react-components/src/components/UnsupportedBrowser.tsx

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/* @conditional-compile-remove(unsupported-browser) */
+import { Icon, Link, Stack, Text } from '@fluentui/react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { _pxToRem } from '@internal/acs-ui-common';
+import React from 'react';
+/* @conditional-compile-remove(unsupported-browser) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(unsupported-browser) */
+import {
+  containerStyles,
+  linkTextStyles,
+  mainTextStyles,
+  secondaryTextStyles
+} from './styles/UnsupportedBrowser.styles';
+
+/**
+ * Strings for UnsupportedBrowser component
+ *
+ * @beta
+ */
+export interface UnsupportedBrowserStrings {
+  /** String for the primary text */
+  primaryText: string;
+  /** String for the secondary text */
+  secondaryText: string;
+  /** String for the help link */
+  moreHelpLink: string;
+}
+
+/**
+ * props for UnsupportedBrowser UI
+ *
+ * @beta
+ */
+export interface UnsupportedBrowserProps {
+  /** Handler to perform a action when the help link is actioned */
+  onTroubleShootingClick: () => void;
+  /** String overrides for the component */
+  strings: UnsupportedBrowserStrings;
+}
+
+/* @conditional-compile-remove(unsupported-browser) */
+const UnsupportedBrowserContainer = (props: UnsupportedBrowserProps): JSX.Element => {
+  const { onTroubleShootingClick, strings } = props;
+  console.log('test2');
+  return (
+    <Stack styles={containerStyles}>
+      <Icon iconName="UnsupportedBrowserWarning"></Icon>
```

Comment:
Icon and text seems a bit too close. 
When looking at the other figma mobile designs, the icon seems to be further up
<img src="https://user-images.githubusercontent.com/97130533/190514998-35db43a3-07b7-433e-b272-ea5d2ccd633c.png" width="25%"/>
<img src="https://user-images.githubusercontent.com/97130533/190515040-53fa12ac-9531-48d5-897e-7cc4ec66a8cb.png" width="25%"/>